src-conf/mssql.sql

```diff
@@ -1327,12 +1327,12 @@ create table workflow_task (
    mod_date datetime null,
    due_date datetime null,
    created_by NVARCHAR(255) null,
-   assigned_to NVARCHAR(255) null,
+   assigned_to NVARCHAR(36) null,
    belongs_to NVARCHAR(255) null,
    title NVARCHAR(255) null,
    description NVARCHAR(MAX) null,
-   status NVARCHAR(255) null,
-   webasset NVARCHAR(255) null,
+   status NVARCHAR(36) null,
+   webasset NVARCHAR(36) null,
```

Comment:
@erickgonzalez @jcastro-dotcms this seems ok. I understand what @erickgonzalez is saying. Only MSSQL is complaining about that matched column-length. We should be fine with this.